package-lock.json

```diff
@@ -3936,9 +3936,9 @@
       }
     },
     "node_modules/caniuse-lite": {
```

Comment:
@beganovich In the last few days, it has constantly been notifying me that this package is outdated, so I decided to update it on this PR. Screenshot:

![Screenshot 2024-04-22 at 19 44 24](https://github.com/invoiceninja/ui/assets/51542191/51ab2db9-f489-4414-874a-1f77b6ed0537)

I hope that's okay from your end as well.